src/cdk/explorer/nodes/getCfnDefinition.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs';
+
+/**
+ * @param uniqueIdentifier unique identifier of state machine
+ * @param cdkOutPath cdk.out path
+ * @param stackName name of parent stack 
+ */
+export function getCfnDefinitionForStateMachine(uniqueIdentifier: string, cdkOutPath: string, stackName: string) {
+
+    try {
+        var data = fs.readFileSync(cdkOutPath + `/${stackName}.template.json`, 'utf8');
+        var jsonObj = JSON.parse(data)
+        var resources = jsonObj.Resources
+
+        for (var key of Object.keys(resources)) {
+            if (key === 'CDKMetadata') continue
+
+            var slicedKey = key.slice(0, -8)
+            if (slicedKey === uniqueIdentifier) {
+                jsonObj = jsonObj.Resources[`${key}`].Properties.DefinitionString["Fn::Join"][1]
+                data = JSON.stringify(jsonObj)
+                data = escape(data)
+                return data
+            }
+        }
+        return 'Wrong state machine identifier'
+    }
+    catch (e) {
+        return 'Unable to get cfn definition for state machine'
+    }
+
+}
+
+function escape(str: string) {
```

Comment:
Each of the variables in this function should have a more descriptive name. Right now we have `str`, `str1`, `str2`, `re1`, `re2`, and none of those describe the purpose of the variable.